RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h

```diff
@@ -130,6 +130,7 @@ class PulseShapeFitOOTPileupCorrection
     double pedSig_;
     double noise_;    
     HcalTimeSlew::BiasSetting slewFlavor_;    
+    const double overlapLimit = 1.0;
```

Comment:
Github doesn't let you attach PDFs, but PNG is okay.  It came through on private email so I'll copy it here as PNGs.


![sbrandt-page2](https://cloud.githubusercontent.com/assets/6480160/5757988/3065b502-9cc1-11e4-8b53-dd4e66c187ab.png)
![sbrandt-page3](https://cloud.githubusercontent.com/assets/6480160/5757989/32b9350e-9cc1-11e4-9c72-2476631880f8.png)
![sbrandt-page4](https://cloud.githubusercontent.com/assets/6480160/5757992/35f5b68e-9cc1-11e4-844a-2861a2bd6502.png)